app/views/pkg/meta_header.mustache

```diff
@@ -0,0 +1,12 @@
+{{! Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+    for details. All rights reserved. Use of this source code is governed by a
+    BSD-style license that can be found in the LICENSE file. }}
+<meta name="twitter:card" content="summary" />
+<meta name="twitter:title" content="{{title}}" />
```

Comment:
See https://developer.twitter.com/en/docs/tweets/optimize-with-cards/guides/getting-started#opengraph – at the bottom

Don't duplicate title/description w/ twitter:

Also, `<meta name="twitter:site" content="@dart_lang" />`